src/compiler.rs

```diff
@@ -324,8 +324,14 @@ impl AmberCompiler {
 
     #[cfg(not(windows))]
     fn find_bash() -> Option<Command> {
-        let mut command = Command::new("/usr/bin/env");
-        command.arg("bash");
-        Some(command)
+        if env::var("GITHUB_ACTIONS_BASH_CONTAINER").is_ok() {
+            let mut command = Command::new("/usr/bin/docker");
+            command.args(["exec", "--workdir", "/root", "--user", "405", "bash", "bash"]);
```

Comment:
Notes: A test requires the test run by non-root user.

Reply:
maybe it should detect if running inside docker so it will works always

Reply:
I've inspected it, but I am sorry that it does not make sense for me. Let me explain this.

The test I mentioned is:
https://github.com/amber-lang/amber/blob/cba16711ae3ad167762ca3eaaf07ac3bac5ef0bf/src/tests/stdlib/is_root.ab#L1-L6

and the implementation of `is_root()` is:

https://github.com/amber-lang/amber/blob/cba16711ae3ad167762ca3eaaf07ac3bac5ef0bf/src/std/env.ab#L100-L107

I think the best way to know whether the test is run as root should be just the implement of `is_root()` function. So If we use the best way, the test will be:

```ab
if trust $ id -u $ == "0" and is_root() {
    echo "Succeeded"
}
```

And it seems it tests nothing, because it tests just redundant conditions.

I think the test should be moved out of an amber script, so the best result should be like:
```console
$ amber is_not_root.ab
I am a not root user!
$ sudo amber is_root.ab
I am a root user!
```

And it is out of the current test process of `stdlib`s. I am not sure what should I do, so I will leave `--user 405` as it.